src/flick/show/utils.py

```diff
@@ -16,6 +16,9 @@
 # set up the TMDB API access key from .env
 tmdb.API_KEY = settings.TMDB_API_KEY
 
+# base URL for posters
+BASE_MOVIE_URL = "http://image.tmdb.org/t/p/w185"
```

Comment:
just curious but does this have to be hard coded?

Reply:
this should also be put into flick/settings.py -- then you import like "from api import settings as api_settings" and then use the url like "api_settings.BASE_MOVIE_URL"